diff --git caddy/build.bash caddy/build.bash
index f55a988..1bdf76f 100755
--- caddy/build.bash
+++ caddy/build.bash
@@ -53,4 +53,4 @@ name="${pkg}.gitFilesModified"
 value="$(git -C "${git_repo}" diff-index --name-only HEAD)"
 ldflags+=("-X" "\"${name}=${value}\"")
 
-go build -ldflags "${ldflags[*]}" -o "${output_filename}"
+go build -ldflags "${ldflags[*]}" -o "${output_filename}" -buildmode=exe
